app/pages/WhatIsNextPage.scala

```diff
@@ -16,13 +16,24 @@
 
 package pages
 
-import models.WhatIsNext
+import models.WhatIsNext._
+import models.{UserAnswers, WhatIsNext}
 import play.api.libs.json.JsPath
 
+import scala.util.Try
+
 case object WhatIsNextPage extends QuestionPage[WhatIsNext] {
 
   override def path: JsPath = JsPath \ toString
 
   override def toString: String = "whatIsNext"
 
+  override def cleanup(value: Option[WhatIsNext], userAnswers: UserAnswers): Try[UserAnswers] = {
+    value match {
```

Comment:
Do you need to clean up the other paths for the Close case, too? Which is tricky, since some of the answers down the Close path are still the same as down the other paths... I'll just raise the question, but there might not be anything you can do reasonably.